app/routes.php

```diff
@@ -84,28 +91,29 @@
         $parsedBody = $request->getParsedBody();
         $url = filter_var($parsedBody['url'], FILTER_SANITIZE_STRING);
         $token = filter_var($parsedBody['token'], FILTER_SANITIZE_STRING);
+
+        $body = apiGet($token, $url);
+        $body = json_encode(json_decode($body), JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
+
+        $settings['accessToken'] = $token;
+        $settings['apiResponse'] = $body;
+        $_SESSION['settings'] = $settings;
+
+        return$container->render($response, 'main.phtml', ["settings" => $settings]);
```

Comment:
I _feel_ bad for pointing this out....but you missed a space.

Reply:
:disappointed: